shared/middleware/helpers/table_coerce.rb

```diff
@@ -0,0 +1,108 @@
+# A set of methods for coercing data in our table to specific types. In general,
+# the goal when converting is to change values where we can, but leave alone
+# those where we don't know how to convert (i.e. I can't convert "123" to a
+# boolean.
+
+module TableCoerce
+  TRUE = /^true$/i
+  FALSE = /^false$/i
```

Comment:
Nice extractions. Maybe do a similar one for the identical regexes in `is_number?` and `to_number` ?